Makefile

```diff
@@ -49,11 +49,18 @@ dynbinary: ## build dynamically linked binary
 plugins: ## build example CLI plugins
 	./scripts/build/plugins
 
-vendor: vendor.conf ## check that vendor matches vendor.conf
+.PHONY: vendor
+vendor: ## update vendor with go modules
 	rm -rf vendor
-	bash -c 'vndr |& grep -v -i clone | tee ./vndr.log'
-	scripts/validate/check-git-diff vendor
-	scripts/validate/check-all-packages-vendored
+	./scripts/vendor update
+
+.PHONY: validate-vendor
+validate-vendor: ## validate vendor
+	./scripts/vendor validate
+
+.PHONY: mod-outdated
+mod-outdated: ## check outdated dependencies
```

Comment:
This is not yet in use, correct? (more of a "local dependabot check")

Reply:
Yes this is only intended for local check

Reply:
Sample output:

```
$ DISABLE_WARN_OUTSIDE_CONTAINER=1 make mod-outdated
./scripts/vendor outdated
+ go-mod-outdated -update -direct
+ go list -mod=vendor -mod=readonly -modfile=vendor.mod -u -m -json all
+--------------------------------------+-----------------------------------------------------+------------------------------------+--------+------------------+
|                MODULE                |                       VERSION                       |            NEW VERSION             | DIRECT | VALID TIMESTAMPS |
+--------------------------------------+-----------------------------------------------------+------------------------------------+--------+------------------+
| github.com/containerd/console        | v1.0.2                                              | v1.0.3                             | true   | true             |
| github.com/containerd/containerd     | v1.5.5                                              | v1.5.8                             | true   | true             |
| github.com/creack/pty                | v1.1.11                                             | v1.1.17                            | true   | true             |
| github.com/docker/docker             | v20.10.3-0.20210811141259-343665850e3a+incompatible | v20.10.12+incompatible             | true   | true             |
| github.com/google/go-cmp             | v0.2.0                                              | v0.5.6                             | true   | true             |
| github.com/mitchellh/mapstructure    | v1.3.2                                              | v1.4.3                             | true   | true             |
| github.com/moby/buildkit             | v0.8.2-0.20210615162540-9f254e18360a                | v0.9.3                             | true   | true             |
| github.com/opencontainers/image-spec | v1.0.1                                              | v1.0.2                             | true   | true             |
| github.com/spf13/cobra               | v1.1.3                                              | v1.3.0                             | true   | true             |
| github.com/tonistiigi/fsutil         | v0.0.0-20210609172227-d72af97c0eaf                  | v0.0.0-20211208191308-f95797418e48 | true   | true             |
| github.com/tonistiigi/go-rosetta     | v0.0.0-20200727161949-f79598599c5d                  | v0.0.0-20201102221648-9ba854641817 | true   | true             |
| golang.org/x/sys                     | v0.0.0-20210823070655-63515b42dcdf                  | v0.0.0-20211216021012-1d35b9e2eb4e | true   | true             |
| golang.org/x/term                    | v0.0.0-20201117132131-f5c789dd3221                  | v0.0.0-20210927222741-03fcf44c2211 | true   | true             |
| golang.org/x/text                    | v0.3.3                                              | v0.3.7                             | true   | true             |
| k8s.io/api                           | v0.16.9                                             | v0.23.0                            | true   | true             |
| k8s.io/apimachinery                  | v0.16.9                                             | v0.23.0                            | true   | true             |
| k8s.io/client-go                     | v0.16.9                                             | v1.5.2                             | true   | true             |
+--------------------------------------+-----------------------------------------------------+------------------------------------+--------+------------------+
```